catroid/src/androidTest/java/org/catrobat/catroid/test/sensing/CollisionInformationTest.java

```diff
@@ -0,0 +1,97 @@
+/*
```

Comment:
We already have a test with exactly this name in a different folder - I would prefer separate names